src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.15.0
+TBD
+
+### Deprecation Notice
+- Deprecate support for Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
+
+### Improvements
+- Add support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md)
+- Upgraded Okteto CLI to [2.5.2](https://github.com/okteto/okteto/releases/tag/2.5.2)
```

Comment:
I think we are mixing present and past tense in the different items. eg Add on the first vs Upgraded on the second. 

Whats the standard to use for release notes?

Reply:
I'm trying to use present tense since it looks more widely used. This is a typo from a copy/pasted. Fixed